saul-core/src/main/scala/edu/illinois/cs/cogcomp/saul/datamodel/node/Node.scala

```diff
@@ -138,7 +145,11 @@ class Node[T <: AnyRef](val keyFunc: T => Any = (x: T) => x, val tag: ClassTag[T
 
   /** Operator for adding a sequence of T into my table. */
   def populate(ts: Iterable[T], train: Boolean = true, populateEdge: Boolean = true) = {
+    progressBar.progress()
+    progressBar.init()
+    progressBar.total = ts.size;
     ts.foreach(addInstance(_, train, populateEdge))
```

Comment:
I don't want to change the original code, that's why I put it in addInstance., I think let's leave it as it is...